### DISCUSSION TITLE
How to make a query to a n:m table with joins?

### DISCUSSION THREAD

**Discussion by pedroarthuralvesdeoliveira** [supabase/supabase#14315] [Questions]
Hey, folks! I've actually read the [docs](https://supabase.com/docs/reference/javascript/select), but I need something more complicated.

I have three tables: phase, process and the n:m table phase_process, which has the primary key from their respective tables. 
The problem is: how can I retrieve phase data through phase_process but inside the process query?   

### The current query
![process-query](https://github.com/supabase/supabase/assets/69775937/9b2d320e-561c-4fd8-ab25-8c095068586d)

### The phase table structure
![phase-table](https://github.com/supabase/supabase/assets/69775937/2a98d165-bf69-4f1c-99a7-312ec28a6c8b)

### The phase_process table structure
![phase_process-table](https://github.com/supabase/supabase/assets/69775937/87a1e98b-f3ff-48db-8ae8-6a50c45031f1)

**Comment by GaryAustin1** [comment]
You should just be able to use `phase(*)` in the select.
